registry-ui/src/pages/Item.tsx

```diff
@@ -457,6 +473,7 @@ const RecordView = observer((props: {}) => {
       {versionControls.render()}
       {revertControls.render()}
       {studyLinkDialog.render()}
+      {renderedDialog}
```

Comment:
all the others are functions - I'm slightly concerned that having it be a constant might be weird - but see how it goes 🤷